jaeger-core/src/test/java/com/uber/jaeger/samplers/HttpSamplingManagerTest.java

```diff
@@ -27,6 +27,7 @@
 import static org.junit.Assert.assertNull;
 
 import com.uber.jaeger.exceptions.SamplingStrategyErrorException;
+import com.uber.jaeger.mocks.MockAgentResource;
```

Comment:
I moved the mock file to a more convenient location so both this and the httpbaggageproxy can both use it.